### PR TITLE
Fixed 186 other postags that were too long

### PR DESCRIPTION
**Pull request by chrisdrymon**
Also changed some underscores to hyphens. The postags that were too long had an extra hyphen at the end. I removed the extra hyphen.

### CODE REVIEW

**Comment by zfletch** [comment]
Thanks for all the work you've put into this PR!

It looks like there's a minor issue with line endings and encodings, making it difficult to review the changes. For example, the entire `public/xml/eur-ba-23-169.xml` file is marked as changed. Would you be able to update the PR to use the original line endings and only change the postags?

**Comment by chrisdrymon** [comment]
That is the only file of all the treebanks that uses that line ending. If you'd like it to conform to the others, you can accept the PR as it is. The line endings were the only thing changed in that file. There were no changes to any postags or anything else in eur-ba-23-169.xml. If you'd like, I can redo it and leave that file alone. But I've already got 4 more commits ahead of this and probably more in the future. And I'd rather not have to work around this one file every time.

**Comment by zfletch** [comment]
It looks like there are two files with Windows line endings: `eur-ba-23-169.xml` and `max-astrol-I-4-1-14.xml`. I think you're correct that it would be a waste of time to have to work around the two files every time somebody wants to make a change. But at the same time, I don't like mixing formatting and content changes in a single commit (or a single PR). It makes the history hard to parse and is a vector for mistakes to get in the repo.

I've pushed a change to just update the formatting. Could you rebase your changes off of the base branch? Then, assuming it looks good to Anise, we can merge the changes.

Thanks again for all your work!

File: public/xml/artem-onir-5.81-5.95.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
       <word id="30" form="ὑμᾶς" lemma="σύ" postag="p2p----a-" relation="OBJ" head="31"/>
       <word id="31" form="ὑποδέξομαι" lemma="ὑποδέχομαι" postag="v1sasm---" relation="OBJ_CO" head="28"/>
       <word id="32" form="," lemma="punc1" postag="u--------" relation="AuxX" head="34"/>
-      <word id="33" form="’" lemma="’" postag="_--------" relation="" head="28"/>
+      <word id="33" form="’" lemma="’" postag="u--------" relation="" head="28"/>
```
Comment (zfletch) [inline public/xml/artem-onir-5.81-5.95.xml:98]: This should be `postag=""` to be consistent: all of the other `form="’"` have `postag=""`.

```
$ grep 'form="’"' *.xml
artem-onir-5.1-5.19.xml:      <word id="15" form="’" lemma="" postag="" relation="AuxX" sg="" head="14"/>
artem-onir-5.20-5.30.xml:      <word id="46" form="’" lemma="" postag="" relation="" sg="" head="39"/>
artem-onir-5.31-5.40.xml:      <word id="37" form="’" lemma="" postag="" relation="" sg="" head="31"/>
artem-onir-5.41-5.50.xml:      <word id="33" form="’" lemma="" postag="" relation="" sg="" head="32"/>
artem-onir-5.41-5.50.xml:      <word id="26" form="’" lemma="" postag="" relation="" sg="" head="25"/>
artem-onir-5.51-5.60.xml:      <word id="34" form="’" lemma="" postag="" relation="AuxX" sg="" head="33"/>
artem-onir-5.71-5.80.xml:      <word id="3" form="’" lemma="" postag="" relation="" sg="" head="0"/>
artem-onir-5.71-5.80.xml:      <word id="17" form="’" lemma="" postag="" relation="AuxX" sg="" head="13"/>
artem-onir-5.81-5.95.xml:      <word id="17" form="’" lemma="" postag="" relation="" head="15"/>
artem-onir-5.81-5.95.xml:      <word id="33" form="’" lemma="’" postag="_--------" relation="" head="28"/>
artem-onir-5.81-5.95.xml:      <word id="25" form="’" lemma="" postag="" relation="" head="22"/>
```

**Comment by chrisdrymon** [comment]
That sounds good. I'll do that.

File: public/xml/herodotus-8-78-83.xml [hunk 2]
```diff
@@ -199,7 +199,7 @@
              span="">
       <word id="1" form="ἡμέας" lemma="ἐγώ" postag="p1p---ma-" relation="OBJ" gloss="" sg="" head="2"/>
       <word id="2" form="στασιάζειν" lemma="στασιάζω" postag="v--pna---" relation="SBJ" gloss="" sg="" head="4"/>
-      <word id="3" form="χρεόν" lemma="χρεών" postag="v-spp_nn_" relation="PNOM" gloss="" sg="" head="4"/>
+      <word id="3" form="χρεόν" lemma="χρεών" postag="x-----n--" relation="PNOM" gloss="" sg="" head="4"/>
```
Comment (zfletch) [inline public/xml/herodotus-8-78-83.xml:202]: This looks correct to me, but I don't think it should be mixed in with the other changes since it's not related to postags that are too long. Maybe a separate PR?

File: public/xml/artem-onir-5.81-5.95.xml (hunk 1 above)
Comment (zfletch) [reply public/xml/artem-onir-5.81-5.95.xml:98]: Actually I think this should go in a separate PR as well since it's not related to postags that are too long or just to changing `_` to `-`. There are a few cases of the `_--------` postag and we should figure out what to do with them:

```
$ grep -R '_--------'
./artem-onir-5.51-5.60.xml:      <word id="7" form="*" lemma="*" postag="_--------" relation="" sg="" gloss="" head=""/>
./artem-onir-5.51-5.60.xml:      <word id="6" form="*" lemma="*" postag="_--------" relation="" gloss="" sg="" head=""/>
./artem-onir-5.61-5.70.xml:      <word id="8" form="*" lemma="*" postag="_--------" relation="" gloss="" sg="" head="10"/>
./artem-onir-5.61-5.70.xml:      <word id="2" form="*" lemma="*" postag="_--------" relation="" gloss="" sg="" head="5"/>
./artem-onir-5.31-5.40.xml:      <word id="8" form="*" lemma="*" postag="_--------" relation="" sg="" gloss="" head="10"/>
./artem-onir-5.31-5.40.xml:      <word id="25" form="*" lemma="*" postag="_--------" relation="" gloss="" sg="" head="20"/>
./artem-onir-5.81-5.95.xml:      <word id="33" form="’" lemma="’" postag="_--------" relation="" head="28"/>
./artem-onir-5.41-5.50.xml:      <word id="12" form="*" lemma="*" postag="_--------" relation="" gloss="" sg="" head=""/>
```
